hello-thirdparty/README.md

```diff
@@ -5,13 +5,16 @@ Hello Third Party Lib is an Android C++ sample that depends on a prebuilt static
 Pre-requisites
 --------------
 - Android Studio 1.3+ with [NDK](https://developer.android.com/ndk/) bundle.
-- [Play Games C++ SDK](https://developers.google.com/games/services/downloads/sdks)
+
+Project structure
```

Comment:
Maybe we could add something less detailed in the description. Since it's part of the sample too: showing you how to manage thirdparty dependencies with a gradle project.